app/models/programBan.py

```diff
@@ -9,3 +9,6 @@ class ProgramBan(baseModel):
     endDate = DateField(null=True)
     banNote = ForeignKeyField(Note, null=False)
     unbanNote = ForeignKeyField(Note, null=True)
+
+    def __str__(self):
+        return f"{self.user.username}"
```

Comment:
`self.user.username` is the same here as just `self.user_id`. I'm nervous that `self.user.username` is running a separate query when it doesn't need to. Not sure if peewee is smart enough or not.

Also, the string representation should include more info. Like the program and/or end date.